packages/safe-kit/src/extensions/messages/SafeMessageClient.ts

```diff
@@ -77,7 +83,7 @@ export class SafeMessageClient {
    * @param {GetSafeMessageListProps} [options] Optional query parameters for pagination
    * @returns {Promise<SafeMessageListResponse>} A list of pending messages
    */
-  async getPendingMessages(options?: GetSafeMessageListProps): Promise<SafeMessageListResponse> {
+  async getPendingMessages(options?: PaginationOptions): Promise<SafeMessageListResponse> {
```

Comment:
Should this function also get an object with optional parameters?

Reply:
apart from the pagination options ? I can only see a parameter to get a concrete message and in this case can be a explicit method. What parameters are you thinking in?